xml/System.Text.Json/JsonReaderOptions.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides the ability for the user to define custom behavior when reading JSON.</summary>
```

Comment:
Is the intention that this would be merged for preview3? I am asking because in the upcoming preview4 build, we moved `MaxDepth` to `JsonReaderOptions` (from `JsonReaderState` here: https://github.com/dotnet/dotnet-api-docs/pull/1880/files#diff-e0eccf9b88a4e527eed98ff46094dbdbR39).

Reply:
Yes, that's my intention 😄